h2o-parsers/h2o-orc-parser/src/test/java/water/parser/ParseTestORCCSV.java

```diff
@@ -72,9 +60,22 @@ public void testParseOrcCsvFiles() {
                 }
             }
 
-        assertTrue(TestUtil.isIdenticalUpToRelTolerance(csv_frame, orc_frame, 1e-5));
+            for (int col_index = 0; col_index < 2; col_index++) {
+                for (int row_index = 0; row_index < orc_frame.numRows(); row_index++) {
+                    long orc = orc_frame.vec(col_index).at8(row_index);
+                    long csv = csv_frame.vec(col_index).at8(row_index);
+                    long diff = orc - csv;
+                    if (abs(diff) > 0) {
+                        Log.info("**orc csv ts differ at row " + row_index + " col " + col_index + " and the difference is "
+                                + Long.toString(diff));
+                    }
+                }
+            }
 
-        csv_frame.delete();
-        orc_frame.delete();
+            assertTrue(TestUtil.isBitIdentical(orc_frame, csv_frame));
+
+            csv_frame.delete();
```

Comment:
You mean I should do scope.enter() at the beginning and do scope.exit() at the end without worrying about frame.delete()?  Let me try that and see.  Thanks for the suggestion, Vlad.